### PR TITLE
Detect MP4 videos

### PR DESCRIPTION
**Pull request by gmetais**
Count the `video/mp4` mime type as a video file.

### CODE REVIEW

File: core/modules/requestsMonitor/requestsMonitor.js [hunk 1]
```diff
@@ -106,6 +106,7 @@ function addContentType(headerValue, entry) {
       break;
 
     case "video/webm":
+    case "video/mp4":
```
Comment (macbre) [inline core/modules/requestsMonitor/requestsMonitor.js:109]: I'm wondering if we can generate a short and small mp4 file (and `webm` too) and use is it in the integration tests.

**Comment by macbre** [comment]
@gmetais - thanks again! :slightly_smiling_face:

**Comment by gmetais** [comment]
Here we are, I've created a wonderful video :)

**Comment by macbre** [comment]
Suspense action thriller :wink:

**Pull request merged**